save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/CookieBanner.kt

```diff
@@ -0,0 +1,52 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package com.saveourtool.save.frontend.components.basic
+
+import com.saveourtool.save.frontend.components.modal.MAX_Z_INDEX
+import com.saveourtool.save.frontend.externals.cookie.acceptCookies
+import com.saveourtool.save.frontend.externals.cookie.cookie
+import com.saveourtool.save.frontend.externals.cookie.declineCookies
+import com.saveourtool.save.frontend.externals.cookie.isAccepted
+import com.saveourtool.save.frontend.externals.i18next.useTranslation
+import com.saveourtool.save.frontend.utils.buttonBuilder
+import com.saveourtool.save.validation.FrontendRoutes
+import js.core.jso
+import react.FC
+import react.VFC
+import react.dom.html.ReactHTML.div
+import react.router.useNavigate
+import react.useState
+import web.cssom.ClassName
+import web.cssom.ZIndex
+
+val cookieBanner: VFC = FC {
+    val (isOpen, setIsOpen) = useState(!cookie.isAccepted())
+    val navigate = useNavigate()
+    val (t) = useTranslation("cookies")
+
+    if (isOpen) {
+        div {
+            className = ClassName("fixed-bottom bg-light px-4 d-flex justify-content-between align-items-center")
+            style = jso {
+                zIndex = (MAX_Z_INDEX - 1).unsafeCast<ZIndex>()
+            }
+            div {
+                className = ClassName("pt-2")
+                markdown("We value your privacy".t().trimIndent())
+            }
+            div {
+                buttonBuilder("Decline".t(), "secondary", classes = "mx-2") {
```

Comment:
do we need allow to decline? maybe use phrase like `You agree with using cookies by continue to use our site`?

Reply:
because we will have to check `cookie.isAccepted()` in all places when we will use cookies

Reply:
> do we need allow to decline? maybe use phrase like `You agree with using cookies by continue to use our site`?

It might break some countries laws :)

Reply:
> because we will have to check `cookie.isAccepted()` in all places when we will use cookies

Yep, we will

But on the other hand, this check is quite light, so it should not cause a lack of performance

Reply:
> > do we need allow to decline? maybe use phrase like `You agree with using cookies by continue to use our site`?
> 
> It might break some countries laws :)

I'm not sure, we notice user and if user agrees, user continues to use the site.

Reply:
for example ![image](https://github.com/saveourtool/save-cloud/assets/1164369/25b493a4-bae9-4376-91a5-493393a5294c)